### ISSUE TITLE
IQKeyboardReturnHandler overrides storyboard settings for return key

### ISSUE THREAD

**Issue by kuemme01** [hackiftekhar/IQKeyboardManager#1418]
**Describe the bug**
I set the return key for the last textfield in view to "Done" because I want to have the blue Done button at the last textfield. But after implementing IQKeyboardReturnHandler the last one are always the default Return button.

**To Reproduce**
1. Build iPad app.
2. Build multiple textfields inside storyboard.
3. Set return key value of every textfield to "Next" except of the last one, set it to "Done".
4. Implement IQKeyboardReturnHandler as described in the docu.
5. Start the app and look at the returnkey of the last textfield.

**Expected behavior**
Last textfield should have a blue "Done" button or for our german keyboard "Fertig".

### Versions
**Xcode:** 9.4.1
**Simulator/Device:** 11.2.6
**Simulator/Device Name:** iPad MH2U2FD/A
**Library Version:** 6.0.4

**Comment by hackiftekhar** [comment]
IQKeyboardReturnHandler is intended to change keyboard return key. Please share a demo project so that I can check.

**Comment by kuemme01** [comment]
Yes, the IQKeyboardReturnHandler does want he should do.
The only problem I have is that he is also overriding settings I have set inside the storyboard (optical settings), the look of the return key.

For the last Textfield I set it to "done", but after implementing IQKeyboardReturnHandler in logic the return key become default one after compiling.

I guess the same happening for the other textfields, IQKeyboardReturnHandler set the return key to "next".
